composer.json

```diff
@@ -11,17 +11,34 @@
             "homepage": "https://github.com/symfony-cmf/ContentBundle/contributors"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/ElectricMaxxx/RestBundle.git"
+        },
+        {
+            "type": "path",
+            "url": "https://github.com/dantleech/Sylius/tree/phpcrodm_form_builder/src/Sylius/Bundle/ResourceBundle",
```

Comment:
@dantleech you know what would be the right solution. Tried several options:
- vcs+ "*.git" to get get sylius, to simply copy => version problem as sylius requires `^1.2`
- path + branch path to ResourceBundle (current one) with your branch to overwrite the "sylius/resource-bundle"  but does not work do not get your branch